### PR TITLE
Add debugging info to container README

### PR DESCRIPTION
**Pull request by jkonecny12**
It's pretty hard to debug tests without this.

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Thank you, this is great!

**Comment by VladimirSlavik** [comment]
On a tangent - not necessarily as part of this PR, but: There isn't any 1-2-3 howto for "_I'm a dev and I want to test my local changes with kickstart tests_". (Or I'm blind and can't see it?) The last step from debugging suggests ```--run-args="--env KSTEST_EXTRA_BOOTOPTS=inst.updates=whatever"```. But there is probably some other way, similar to how the PR test is executed. It might be useful to have that explicitly spelled out somewhere, as that is a primary reason to run a non-scheduled kickstart test.

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

File: containers/runner/README.md [hunk 1]
```diff
@@ -88,3 +85,30 @@ To stop the proxy, call
...
+
+     ************************************************************************
+     You can connect to this container's libvirt with this connection string:
+    
+        qemu+tcp://${MY_IP}/session
+    
+     ************************************************************************
+     
+The last thing is to take the ``qemu+tcp://<IP>/session`` and put that to virtual 
+machine manager (``virt-manager``). Then you can see and control VM by the manager.
```
Comment (poncovka) [inline containers/runner/README.md:111]: You can run `virt-manager -c qemu+tcp://${MY_IP}/session` to add the new connection to the virtual manager.

Comment (VladimirSlavik) [reply containers/runner/README.md:111]: That's actually pretty helpful, the normal GUI way requires a choice (Hypervisor = Custom URI) that is somewhat non-obvious... I assume this also opens the window?

Comment (jkonecny12) [reply containers/runner/README.md]: Good point. Added.

**Review by VladimirSlavik — APPROVED** [review]
Thank you!

**Review by poncovka — APPROVED** [review]
It looks great!

**Pull request merged**